src/main/java/ch/epfl/biop/imaris/EasyXT.java

```diff
@@ -774,6 +782,85 @@ public static IDataSetPrx getSurfacesDataset(ISurfacesPrx surface, double downsa
         return data;
     }
 
+    /**
+     * Get an ImagePlus of the spots as a mask (255)
+     *
+     * @param spots , a spots object see {@link  #getSpots(String)}
+     * @return , an ImagePlus
+     * @throws Error
+     */
+    public static ImagePlus getSpotsMask(ISpotsPrx spots) throws Error {
+        return getSpotsImage(spots, false, false);
+    }
+
+    // TODO Comment
+    public static ImagePlus getSpotsLabel(ISpotsPrx spots) throws Error {
+        return getSpotsImage(spots, true, false);
+    }
+
+    // TODO Comment
+    public static ImagePlus getSpotsImage(ISpotsPrx spots, boolean isValueID, boolean isGauss) throws Error {
+        // Prepare the imp to receive spots pixels
+        //
+        // get the calibration info from Imaris
+        ImarisCalibration cal = new ImarisCalibration(app.GetDataSet());
+        // Create a new imp with the cal and get stack ()
+        //ImagePlus imp = IJ.createHyperStack( getOpenImageName(), cal.xSize , cal.ySize, cal.cSize, cal.zSize, cal.tSize , 32);
+        //imp.setCalibration( cal );
+        //ImageStack stack = imp.getStack();
+
+        long[] spots_ids = spots.GetIds();
+        float[][] spots_centerXYZ = spots.GetPositionsXYZ();
+        float[][] spots_radiiXYZ = spots.GetRadiiXYZ();
+        int[] spots_t = spots.GetIndicesT();
+
+        double resXY = cal.pixelWidth;
+        double resZ = cal.pixelDepth;
+        String unit = cal.getUnit();
+
+        // Define default vectors
+        Vector3D V = new Vector3D(1, 0, 0);
+        Vector3D W = new Vector3D(0, 1, 0);
+        if (Math.abs(V.dotProduct(W)) > 0.001) {
```

Comment:
The vectors are defined manually above, no need for this sanity check

Reply:
Left it from the original [Shape3D_.java](https://github.com/mcib3d/mcib3d-plugins/blob/master/src/main/java/mcib_plugins/Shape3D_.java) paste in case we modify them later.